build/build-image/Dockerfile

```diff
@@ -39,17 +39,23 @@ RUN wget -q https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.zip && un
 ENV PATH /usr/local/go/bin:/go/bin:/opt/google-cloud-sdk/bin:$PATH
 
 # RUN gcloud components update
-RUN gcloud components update && gcloud components install kubectl
+RUN gcloud components update
+
+# install kubectl without gcloud as we want the last version
+ENV KUBECTL_VER 1.11.0
```

Comment:
Super minor thing. Do we want 1.10? I don't think you can run 1.11 on anything yet?

Reply:
Good question ! I was lazy to figured which version I need.

I updated kubectl because the version from gcloud doesn't support port-forward on stateful set and I needed it there : 

https://github.com/GoogleCloudPlatform/agones/blob/dc0e9dd3df062e58366320f44a932c1931b65d8a/build/e2e-image/entrypoint.sh#L29

it has been introduced by https://github.com/kubernetes/kubernetes/pull/59705 which seems to be 1.10, do you want me to downgrade ?

Reply:
I think so - it's super minor, bit it's nice to have our supported version line up with our Dev tooling, Just to make sure nothing odd happens.

Other than that - swash to a single commit, and I'll got approve!